.github/workflows/deploy.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-latest, macos-latest]
```

Comment:
Umm, where's "ubuntu-latest"? It is listed in current `main`: https://github.com/pytroll/python-geotiepoints/blob/main/.github/workflows/ci.yaml#L12

Reply:
Oh, that was `deploy.yaml`, not `ci.yaml`.